cpp/CMakeLists.txt

```
@@ -160,7 +160,10 @@ target_link_libraries(raft INTERFACE
         CUDA::cudart
         CUDA::cusparse
         rmm::rmm
+<<<<<<< Updated upstream
         $<$<BOOL:${RAFT_ENABLE_cuco_DEPENDENCY}>:cuco::cuco>
+=======
+>>>>>>> Stashed changes
```

```suggestion
        $<$<BOOL:${RAFT_ENABLE_cuco_DEPENDENCY}>:cuco::cuco>
```

Remnants from git merge

---

Lol, yep CI just informed me